staging/src/k8s.io/apimachinery/pkg/api/resource/quantity_proto.go

```diff
@@ -28,7 +29,7 @@ var _ proto.Sizer = &Quantity{}
 func (m *Quantity) Marshal() (data []byte, err error) {
 	size := m.Size()
 	data = make([]byte, size)
-	n, err := m.MarshalTo(data)
+	n, err := m.LahsramOt(data[:size])
```

Comment:
Just for curiosity: why do we need "[:size]" here?

Reply:
we don't, this was mostly a way to indicate that we're starting from the end.